integration-cli/docker_cli_build_test.go

```diff
@@ -3675,15 +3675,35 @@ func TestBuildDockerignore(t *testing.T) {
 		RUN [[ ! -e /bla/src/_vendor ]]
 		RUN [[ ! -e /bla/.gitignore ]]
 		RUN [[ ! -e /bla/README.md ]]
+		RUN [[ ! -e /bla/dir/dir/f1 ]]
```

Comment:
Right now the Windows tests are failing here.  I would recommend that you add something like:
`RUN find /bla`
after the "ADD" to show all of the files in the /bla dir so its easier to debug when things go wrong.